### PR TITLE
infoschema: add missing columns in collations_information_applicability

### PR DESCRIPTION
**Pull request by zhexuany**
In mysql, the output is the following:

```
show create table COLLATION_CHARACTER_SET_APPLICABILITY;
+---------------------------------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
| Table                                 | Create Table                                                                                                                                                                                                     |
+---------------------------------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
| COLLATION_CHARACTER_SET_APPLICABILITY | CREATE TEMPORARY TABLE `COLLATION_CHARACTER_SET_APPLICABILITY` (
  `COLLATION_NAME` varchar(32) NOT NULL DEFAULT '',
  `CHARACTER_SET_NAME` varchar(32) NOT NULL DEFAULT ''
) ENGINE=MEMORY DEFAULT CHARSET=utf8 |
+---------------------------------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
1 row in set (0.01 sec)
```

In tidb, the output is the following:
```
mysql> show create table COLLATION_CHARACTER_SET_APPLICABILITY;
+---------------------------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
| Table | Create Table |
+---------------------------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
| COLLATION_CHARACTER_SET_APPLICABILITY | CREATE TABLE `COLLATION_CHARACTER_SET_APPLICABILITY` (
 `TABLESPACE_NAME` varchar(64) DEFAULT NULL,
 `ENGINE` varchar(64) DEFAULT NULL,
 `TABLESPACE_TYPE` varchar(64) DEFAULT NULL,
 `LOGFILE_GROUP_NAME` varchar(64) DEFAULT NULL,
 `EXTENT_SIZE` int(21) UNSIGNED DEFAULT NULL,
 `AUTOEXTEND_SIZE` int(21) UNSIGNED DEFAULT NULL,
 `MAXIMUM_SIZE` int(21) UNSIGNED DEFAULT NULL,
 `NODEGROUP_ID` int(21) UNSIGNED DEFAULT NULL,
 `TABLESPACE_COMMENT` varchar(2048) DEFAULT NULL
) ENGINE=InnoDB DEFAULT CHARSET=utf8 COLLATE=utf8_bin |
+---------------------------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
1 row in set (0.00 sec)
```

This PR fixed inconsistent behavior.  I am not sure how to test this change since I did not find any related test protocol for similar code pattern. 
@tiancaiamao @zz-jason PTAL

### CODE REVIEW

**Comment by zz-jason** [comment]
@zhexuany The example in description is a little complex, can you take another simpler one to show the inconsistent behavior exactly?

**Comment by zhexuany** [comment]
```
In mysql:
use information_schema;
select count(COLLATION_NAME) from COLLATION_CHARACTER_SET_APPLICABILITY;
+-----------------------+
| count(COLLATION_NAME) |
+-----------------------+
|                   222 |
+-----------------------+
1 row in set (0.01 sec)

In TiDB:
use information_schema;
select count(COLLATION_NAME) from COLLATION_CHARACTER_SET_APPLICABILITY;
ERROR 1054 (42S22): Unknown column 'collation_name' in ''

```

**Comment by shenli** [comment]
LGTM

File: executor/aggregate_test.go [hunk 1]
```diff
@@ -243,7 +243,7 @@ func (s *testSuite) TestAggregation(c *C) {
 
 	result = tk.MustQuery("select count(*) from information_schema.columns")
 	// When adding new memory table in information_schema, please update this variable.
-	columnCountOfAllInformationSchemaTables := "743"
```
Comment (zimulala) [inline executor/aggregate_test.go]: It's column count, do we need to update the comment in line244?

**Comment by zhexuany** [comment]
@shenli @zimulala PTAL

**Pull request merged**